django_app/pyproject.toml

```diff
@@ -43,6 +43,7 @@ django-allauth = "^0.61.1"
 django-otp = "^1.3.0"
 humanize = "^4.9.0"
 yarl = "^1.9.4"
+furl = "^2.1.3"
```

Comment:
re-added and updated version of furl

Reply:
Sorry - I think this was my mistake - but furl is a huge foot-pistol. yarl is far, far better better, because its URL instances are immutable.